encrypted-media-respec.html

```diff
@@ -2332,8 +2332,8 @@
         <section>
           <h4><a>MediaKeyMessageEventInit</a></h4>
           <div><pre class="idl">[SecureContext] dictionary MediaKeyMessageEventInit : EventInit {
-             MediaKeyMessageType messageType = "license-request";
-             ArrayBuffer         message;
+             required MediaKeyMessageType messageType;
```

Comment:
`"license-request"` is the most obvious one, but it's not really a "safe" default. It seems better to make the author specify it along with the message.

I don't expect anyone is actually using this directly.